packages/lodestar-validator/src/util/httpClient.ts

```diff
@@ -17,8 +18,9 @@ export class HttpClient {
     this.logger = logger;
   }
 
-  public async get<T>(url: string): Promise<T> {
+  public async get<T>(url: string, query?: querystring.ParsedUrlQueryInput): Promise<T> {
```

Comment:
What do you think about passing query object instead of ParsedUrlQueryInput?
 That way we could easily switch how we serialize query.